angular/src/app/screens/sign-up/sign-up.component.ts

```diff
@@ -1,25 +1,41 @@
 import { HttpResponse } from '@angular/common/http';
 import { Component } from '@angular/core';
-import { SIGN_UP_FIELDS } from '../../constants/formAccount';
+import { SIGN_UP_FIELDS } from '../../constants/form-account';
 import { IUser } from 'src/app/interfaces/global.interface';
 import { UserService } from '../../services/user.service';
+import { StatusRequest } from 'src/app/constants/code-request';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { CustomValidators } from 'src/app/helpers/utilities/custom-validators';
 
 @Component({
   selector: 'app-sign-up',
   templateUrl: './sign-up.component.html',
-  styleUrls: ['./sign-up.component.scss'],
 })
 export class SignUpComponent {
   fields = SIGN_UP_FIELDS;
   buttonSend = 'Sign Up';
 
   constructor(private userService: UserService) {}
 
+  formControls = new FormGroup(
```

Comment:
You don't have a `FormControl` array, you have a variable `FormGroup` type, you can call it `formGroup` or just `form`.